### PR TITLE
Feature/features cards

### PR DESCRIPTION
**Pull request by ngocphong230801**
In this pull request :
- Add code for features-cards of index.
- Add CSS  for feature-cards.

### CODE REVIEW

File: practices/practice-one/src/styles/layout/feature-card.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    margin-top: 40px;
+}
+
+.features-cards .feature-card:nth-child(3) {
+    margin-top: 80px;
+}
+
+.feature-card .feature-card-top {
+    display: flex;
+    background-color: #f3f7ff;
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/styles/layout/feature-card.css]: any color variable?

File: practices/practice-one/src/index.html [hunk 2]
```diff
@@ -131,6 +131,64 @@ <h3 class="heading-title">How VanGoux works</h3>
...
+                        <div class="feature-card-bottom">
+                            <p class="feature-card-bottom-content">Crisp, pixel-pefect landing pages without bothering your 
+                            developers</p>
+                            <div class="card-decoration">
+                                <img src="../src/assets/img/DecorationJuju.png" alt="Decoration">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="feature-card">
+                        <div class="feature-card-top">
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/index.html]: don't use top/bottom for class name. If position changes, you have to update name

File: practices/practice-one/src/index.html [hunk 3]
```diff
@@ -131,6 +131,64 @@ <h3 class="heading-title">How VanGoux works</h3>
...
+                            <div class="feature-card-top-icon">
+                                <img src="../src/assets/icon/IconSolidFlexibleResponsive.png" alt="IconSolidFlexibleResponsive">
+                            </div>
+                            <div class="feature-card-top-content">
+                                <p class="features-cards-title">Flexible & responsive</p>
+                            </div>
+                        </div>
+                        <div class="feature-card-bottom">
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/index.html]: ditto

File: practices/practice-one/src/index.html [hunk 4]
```diff
@@ -131,6 +131,64 @@ <h3 class="heading-title">How VanGoux works</h3>
...
+                    <h3 class="heading-title">Features</h3>
+                </div>
+                <div class="features-cards">
+                    <div class="feature-card">
+                        <div class="feature-card-top">
+                            <div class="feature-card-top-icon">
+                                <img src="../src/assets/icon/IconSolidPixelPerfect.png" alt="IconSolidPixelPerfect">
+                            </div>
+                            <div class="feature-card-top-content">
+                                <p class="features-cards-title">Pixel perfect output</p>
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/index.html]: it's in `feature-card`, why class name is `features-cards-title` here

File: practices/practice-one/src/styles/layout/feature-card.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
+.heading-title:nth-child(2) {
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/layout/feature-card.css]: Should styled in heading, if use in card component, please add more class for it

File: practices/practice-one/src/styles/layout/feature-card.css [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    font-size: var(--font-size-text-xl);
+    font-weight: var(--font-weight-light);
+    line-height: var( --line-height-sm);
+    width: 159px;
+    padding: 10px 0;
+    margin: 0;
+    position: absolute;
+    top: 20px;
+    left: 139px;
+
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/layout/feature-card.css]: code style guide

**Review by huyenpham-agilityio — APPROVED** [review]

**Pull request merged**